controllers/review-status/review-status.js

```diff
@@ -199,12 +212,12 @@ function sendResponse(passon) {
             reject(error);
             return;
         } else {
-            if (process.startsWith('Blind')) {
-                passon.reviewStatus = 'Blind';
-            } else if (process.startsWith('Double blind')) {
-                passon.reviewStatus = 'Double blind';
+            if (process.toLowerCase().startsWith('blind')) {
```

Comment:
There seems to be no documentation on the different values used. I just found this at https://libraries.io/github/DOAJ/doaj:

```
        "editorial_review" : {
            "process" : "<type of editorial review process>",
            "url" : "<url to info about editorial review process>"
        }
```